src/platform/assets/components/UploadModelUpgradeModal.vue

```diff
@@ -0,0 +1,39 @@
+<template>
+  <div
+    class="upload-model-upgrade-modal flex flex-col justify-between gap-10 p-4 border-t-[1px] border-border-default"
```

Comment:
I often forget this one too.

```suggestion
    class="upload-model-upgrade-modal flex flex-col justify-between gap-10 p-4 border-t border-border-default"
```